Sources/DatadogObjc/DatadogConfiguration+objc.swift

```diff
@@ -16,6 +16,18 @@ public class DDEndpoint: NSObject {
     }
 
     // MARK: - Public
+    
+    @objc
+    public static func us1() -> DDEndpoint { .init(sdkEndpoint: .us1) }
+    
+    @objc
+    public static func us3() -> DDEndpoint { .init(sdkEndpoint: .us3) }
+    
+    @objc
+    public static func eu1() -> DDEndpoint { .init(sdkEndpoint: .eu1) }
+    
+    @objc
+    public static func us1_fed() -> DDEndpoint { .init(sdkEndpoint: .us1_fed) }
```

Comment:
we have `DDLogsEndpoint` / `DDTracesEndpoint` but we don't have `DDRUMEndpoint`, did we forget it or should we deprecate `DDLogsEndpoint` / `DDTracesEndpoint` as well?

Reply:
Both `set(logsEndpoint: DDLogsEndpoint)` and `public func set(tracessEndpoint: DDLogsEndpoint)` were deprecated when introducing common `func set(endpoint: DDEndpoint)`. Enum cases were not deprecated, but they could be.